CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Up-For-Grabs Code of Conduct
+
+This document outlines how contributors to the Up-For-Grabs project
+should interact with each other, and the process for resolving issues.
+
+Projects listed on Up-For-Grabs are not covered by this code of conduct.
+Inquiries about a specific project should be addressed according to that
+project's governance rules.
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
```

Comment:
```suggestion
Unacceptable behavior by participants includes, but is not limited to:
```

Reply:
This is already superfluous to how project maintainers are meant to react to "any unacceptable behaviour" below.
However, that is only an expectancy.
"Unacceptable" thus functionally maintains parity with "acceptable".
The only "responsibility" is to instrument censorship for anything that isn't aligned to the CoC, or for "behaviors that they deem inappropriate, threatening, offensive, or harmful."
That is a police state with the law above the people, and its enactor. Made worse in the newer version btw.
In it the least amount of vested interest is tied to the same vagueness and whim, with zero oversight.